webpack.dev.js

```diff
@@ -0,0 +1,10 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+    mode: 'development',
+    devtool: 'inline-source-map',
+    devServer: {
+        contentBase: './dist',
```

Comment:
I'm not sure you need this, since you don't need to serve static files yet, and if you would it's gonna be "public" folder.

Reply:
done